addons/sale_pdf_quote_builder/models/ir_actions_report.py

```diff
@@ -158,15 +158,16 @@ def _get_sol_form_fields_mapping(self, line):
         """
         line.ensure_one()
         env = self.with_context(use_babel=True).env
+        formatted_sale_price = u''
```

Comment:
you can remove the u, it's equivalent but you don't need in in current version of odoo